creator-node/src/routes/prometheusMetricsExporter.js

```diff
@@ -0,0 +1,22 @@
+const Prometheus = require('prom-client')
+
+const { handleResponse } = require('../apiHelpers')
+const PrometheusMetric = require('../services/PrometheusMetric')
+
+const collectDefaultMetrics = Prometheus.collectDefaultMetrics
+const metricPrefix = 'audius_cn_'
+
+collectDefaultMetrics({ prefix: metricPrefix })
+
+module.exports = function (app) {
+  app.get('/prometheus_metrics', async (req, res) => {
```

Comment:
1. did you intentionally not want to use `handleResponse` and `sendResponse` per our route pattern?
2. does it make sense to add rate limits to this function

Reply:
looks like you took this code from this example from their docs (which is not very good btw) https://github.com/siimon/prom-client/blob/master/example/server.js#L102-L105

since this is not part of our core API and purely additive i'm open to either way as long as we document why it's different but i agree with sid that we should try to use our function for this if possible

Reply:
I gave it a shot and was able to use `handleResponse`, `errorResponseServerError`, and `sendResponse`.

I do have to set `res.set('Content-Type', Prometheus.register.contentType)`, but hopefully that shouldn't cause any issues for this one request path.